ambry-test-utils/src/main/java/com/github/ambry/router/InMemoryRouter.java

```diff
@@ -391,195 +398,175 @@ protected static void completeOperation(FutureResult futureResult, Callback call
       callback.onCompletion(operationResult, exception);
     }
   }
-}
-
-/**
- * Thread to read the post data async and store it.
- */
-class InMemoryBlobPoster implements Runnable {
```

Comment:
The diff looks weird in the class, but the change I made is to change the package-private classes below InMemoryRouter in this file into private static inner classes, since they are not used anywhere outside of InMemoryRouter